### ISSUE TITLE
Child vs Adult routing

### ISSUE THREAD

**Issue by Zannick** [brakkum/ZOoTR-Entrance-Tracker#6]
All the individual "Ignore" settings are great, but repeatedly toggling them back and forth as I switch between child and adult is tedious. A mutually-exclusive "Consider as child/adult/both" would be nice.

Along these lines, Bolero of Fire is only usable as child to reach the center platform in the Crater, and it doesn't have an "Ignore" setting other than the BoF collector at the bottom, which presented trouble when routing where the only valid option was a different song (so hitting "Ignore Songs" would say the route's not available yet).

**Comment by brakkum** [comment]
I agree, the routing leaves a lot to be desired for very specific situations. Song specific things especially can be tedious. This is something I’ve considered implementing in a newer version. I haven’t worked on the tracker in quite awhile, but might pick up a new build as something to do in my free time. Will keep you in the loop!
